integration_tests/steps/integration_tests.py

```diff
@@ -728,35 +709,6 @@ def test_accumulated_energy(context):
     assert isclose(net_energy, 0, abs_tol=1e-10)
 
 
-@then('the energy bills report the correct external traded energy and price')
```

Comment:
I think it is wrong to remove it now. Keeping the TODO there is important to not forget to reactivate it once the bug is resolved.

Reply:
IMO this test is totally outdated. There is no bug but another way of calculation. Shall I add another story to add  a real `the energy bills report the correct external traded energy and price` test?

Reply:
Why not adding a link to the 2-sided PAC bug ticket? That way we know that this needs to be rewritten when the bug is resolved.

Reply:
Can you please provide the link to the ticket ? I cant find it.
So I leave the implementation here but link the ticket, yes?

Reply:
(also disabling this step again, of course)

Reply:
https://gridsingularity.atlassian.net/browse/D3ASIM-1887 , it is part of the TODO comment on the step implementation. I had a deeper look in the test, and I cannot understand why the test is outdated. I think it is perfectly valid still, and it essentially checks out if the trade_energy_price graph is consistent with the energy bills, as far as I understand. The only reason why this step should fail, is due to the behavior described in the bug. Why is it important to delete it actually ?

Reply:
Ok, now it makes sense, thanks for charing the ticket!
Will re add the step again. Thanks!